tests/test_compound.py

```diff
@@ -696,6 +696,62 @@ def test_autocorrelate():
     print 'time for %s = %.2f'%(funcname(),t2-t1)
 
 
+def test_fourier_sqrt():
+    """Test that the FourierSqrt operator is the inverse of auto-convolution.
+    """
+    import time
+    t1 = time.time()
+
+    dx = 0.4
+    myImg1 = galsim.ImageF(80,80, scale=dx)
+    myImg1.setCenter(0,0)
+    myImg2 = galsim.ImageF(80,80, scale=dx)
+    myImg2.setCenter(0,0)
+
+    # Test trivial case, where we could (but don't) collapse the chain of SBProfiles
+    # by recognizing the inverse.
```

Comment:
Actually, here I meant "recognize that FourierSqrt is the inverse of AutoConvolve"; I'll expand the comment to clarify that.